deploy/template.yaml

```diff
@@ -56,6 +56,10 @@ Globals:
           - IsDevelopment
           - 0
           - 3
+        OAUTH_CRI_ENC_KEY_CACHE_DURATION_MINUTES: !If
```

Comment:
This could probably just be 'standard' config - i.e. using an SSM parameter, unless there's a good reason to make it static?

I'd also suggest just having one value for `OAUTH_KEY_CACHE_DURATION`, I don't think we're likely to need different values for the oauth key types.

Reply:
PR for this is here: https://github.com/govuk-one-login/ipv-core-common-infra/pull/1109/files